### PR TITLE
[Rails 6.0] Fix order adjustments spec

### PR DESCRIPTION
**Pull request by andrewpbrett**
#### What? Why?

Paging @Matt-Yorkley to take a look at this... I'm a bit puzzled as to what would have changed in Rails 6 to flip this spec to red. Let's see if the fix works on the current master. 

#### What should we test?

Green build 

#### Release notes

Updated an Order spec to prepare for Rails 6

Changelog Category: Technical changes

### CODE REVIEW

**Review by luisramos0 — APPROVED** [review]
Could it be related to autoloading?
https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#autoloading

**Comment by Matt-Yorkley** [comment]
What was the error? It's difficult to review without the context.

File: spec/models/spree/order_spec.rb [hunk 1]
```diff
@@ -1141,12 +1141,9 @@
...
 
       context "when finalized fee adjustments exist on the order" do
-        let(:payment_fee_adjustment) { order.all_adjustments.payment_fee.first }
-        let(:shipping_fee_adjustment) { order.shipment_adjustments.first }
-
         before do
-          payment_fee_adjustment.finalize!
-          shipping_fee_adjustment.finalize!
+          order.all_adjustments.payment_fee.each(&:finalize!)
+          order.shipment_adjustments.each(&:finalize!)
```
Comment (Matt-Yorkley) [inline spec/models/spree/order_spec.rb:1146]: We could maybe just do `order.all_adjustments.each(&:finalize!)` here for simplicity. Not a big deal though.

**Comment by andrewpbrett** [comment]
@Matt-Yorkley you can see it in this build: https://github.com/openfoodfoundation/openfoodnetwork/runs/2536112257?check_suite_focus=true

```
2) Spree::Order a completed order with shipping and transaction fees removing line_items when finalized fee adjustments exist on the order does not attempt to update such adjustments
     Failure/Error: expect(order.adjustment_total).to eq expected_fees
     
       expected: 16
            got: 0.13e2
     
       (compared using ==)
     # ./spec/models/spree/order_spec.rb:1158:in `block (5 levels) in <top (required)>'
```

From what I could tell the math it's doing it 2*5 + 1*3, when it should be doing 2*(5+3) (see line 1118 of the spec file).

**Comment by Matt-Yorkley** [comment]
Alright, cool. Seems like it's an issue with the test setup as opposed to the code itself :ok_hand:

**Review by Matt-Yorkley — APPROVED** [review]
:+1:

**Pull request merged**